gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/TraversalEvaluation/TraversalParser.cs

```diff
@@ -398,7 +398,7 @@ private static ITokenParameter ParseParameter(string text, ref int i)
             {
                 return ParseNumber(text, ref i);
             }
-            if (text.Substring(i, 3).StartsWith("__."))
+            if (text.Length >= i + 3 && text.Substring(i, 3).StartsWith("__."))
```

Comment:
Not a big deal, but with the length check included, it can be an equality check, rather than `StartsWith()`.

Reply:
Done